chia/cmds/plotnft_funcs.py

```diff
@@ -181,6 +181,29 @@ async def pprint_pool_wallet_state(
             print(f"Expected to leave after block height: {expected_leave_height}")
 
 
+async def pprint_all_pool_wallet_state(
```

Comment:
Extra p in print

Reply:
Wait isn't this function above?

Reply:
`pprint` means pretty-print.
There is alerady `pprint_pool_wallet_state` function around line 122.
`pprint_all_pool_wallet_state` just runs `pprint_pool_wallet_state` through all pool wallet.

Reply:
pprint is still kinda stupid but sounds good.